packages/material-ui/src/InputAdornment/InputAdornment.js

```diff
@@ -16,10 +16,11 @@ export const styles = {
   },
   /* Styles applied to the root element if `variant="filled"`. */
   filled: {
-    '&$positionStart': {
+    '&$positionStart:not($marginDense):not($hiddenLabel)': {
       marginTop: 16,
     },
   },
+
```

Comment:
```suggestion
```

Reply:
Just  a whitespace artifact from transpilation and accepted by prettier.

Reply:
Could we remove it for consistency?

Reply:
To expand on this point. I believe that prettier main value comes from its help in increasing the consistency on the codebase. It comes with an opinionated coding style. People quickly realized that the actual format doesn't really matter, people have stopped arguing: "should I use a space or not?" and follow the tool. I think that we should continue to aim for maximizing code consistency using the following ordered criterions 1. Prettier when raising his voice, 2. The existing codebase majority, 3. Fallback to our taste.

What do you think?